.changeset/config.json

```diff
@@ -7,5 +7,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["*nextjs-app-router*", "*nextjs-pages-router*"]
```

Comment:
Not sure how the heck this is working for them.

![CleanShot 2024-02-08 at 11 37 00@2x](https://github.com/makeswift/makeswift/assets/20950876/e015d254-f121-4e21-9494-d3968d5eb9da)

In the source code they are doing ^ and not using micromatch glob expressions like they are supposed to, and 
![CleanShot 2024-02-08 at 11 38 35@2x](https://github.com/makeswift/makeswift/assets/20950876/8ffb474e-192d-41d1-9988-0468384ee88b)

I am removing the * since they aren't helping anyways

Reply:
Regardless in the version we were on `2.22.0` the `ignore` array wasn't being checked for the `add` command. That was the root issue. It was added in `2.25.1`